File: Core/src/main/java/com/hqq/core/annotation/LayoutModel.java
package com.hqq.core.annotation;

import android.support.annotation.IntDef;

import com.hqq.core.ui.RootViewBuild;


/**
 * @Author : huangqiqiang
 * @Package : com.shangwenwan.sww.annotation
 * @FileName :   LayoutModel
 * @Date : 2019/1/17 0017  下午 1:43
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
@IntDef({LayoutModel.LAYOUT_MODE_LINEAR_LAYOUT, LayoutModel.LAYOUT_MODE_FRAME_LAYOUT})
public @interface LayoutModel {
    /**
     * 布局
     * LinearLayout
     */
    int LAYOUT_MODE_LINEAR_LAYOUT = 1;
    /**
     * 布局
     * frameLayout
     */
    int LAYOUT_MODE_FRAME_LAYOUT = 2;
}

File: Core/src/main/java/com/easy/core/annotation/LayoutModel.kt
package com.easy.core.annotation

import androidx.annotation.IntDef


/**
 * @Author : huangqiqiang
 * @Package : com...annotation
 * @FileName :   LayoutModel
 * @Date : 2019/1/17 0017  下午 1:43
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
@IntDef(LayoutModel.LAYOUT_MODE_LINEAR_LAYOUT, LayoutModel.LAYOUT_MODE_FRAME_LAYOUT)
annotation class LayoutModel {
    companion object {
        /**
         * 布局
         * LinearLayout
         */
        const val LAYOUT_MODE_LINEAR_LAYOUT = 1

        /**
         * 布局
         * frameLayout
         */
        const val LAYOUT_MODE_FRAME_LAYOUT = 2
    }
}

